runtime/onert/backend/acl_common/Swizzle.h

```diff
@@ -45,6 +45,21 @@ class ARMComputeAxis
   uint32_t _value;
 };
 
+inline int ARMCompareLayouts(uint32_t rank, const ir::Layout org_layout = ir::Layout::UNKNOWN,
```

Comment:
Please, try to use ToARMComputeAxis instead of introducing new function.

Reply:
Applied the change, thanks for the comment.